scss/tech.scss

```diff
@@ -60,3 +47,8 @@ $tech-logo-size: 100px;
 .tech-title {
   font-size: $text-medium;
 }
+
+.tech-col {
+  grid-area: techcol;
```

Comment:
As you wont have template-areas any more, this could be grid-column: 1, grid-row: 1/3